timescaledb/overview/release-notes/index.md

```diff
@@ -33,19 +32,33 @@ functionality works, and, as a result, made APIs simpler and more consistent,
 while also empowering users with more control and flexibility to customize
 behaviors to suit your needs.  Some of these API updates are **breaking changes**.
 
-What's new in TimescaleDB 2.1:
+What's new in TimescaleDB 2.2.1:
 
-- [Support for PostgreSQL 13.2](https://github.com/timescale/timescaledb/issues/2779):
-PG13.2 offers various improvements in functionality, administration, and security,
-among others. Also, as a result of our own @svenklemm upstream report of
-CVE-2021-20229 on PostgreSQL, we have waited for the release of PG13.2
-(released Feb 11, 2021) to provide the safest experience to our users.
-- Support for [ALTER/RENAME columns for compressed hypertables](https://github.com/timescale/timescaledb-private/issues/849), 
-which simplifies the user journey if a user wants to change the schema and
-they have already compressed their hypertables. 
+Skip Scan optimization on single node and multinode, which significantly 
+improves the performance of queries with DISTINCT ON. This 
+optimization is not yet available for queries on distributed 
+hypertables.
+
+This release also adds a function to create a distributed 
+restore point, which allows performing a consistent restore of a 
+multi-node cluster from a backup.
```

Comment:
Nit: We are in general writing too long sentences in our documentation. I prefer terse and distinct text.

```suggestion
This release also adds support for creating a distributed 
restore point. This allows performing a consistent restore of a 
multi-node cluster from a backup.
```